_posts/24-08-2015-caveats-with-ruby-interrupt.md

```diff
@@ -0,0 +1,263 @@
+---
+language: english
+layout: post
+comments: true
+title: 'Caveats interrupting Ruby code'
+---
+
+# <p hidden>caveats-with-ruby-interrupt<p hidden>
```

Comment:
It's because org-mode. I want to have a top level heading that I can fold, but I don't want it to show in the html.